web/src/api/getAppVersion.ts

```diff
@@ -13,7 +13,6 @@ async function getVersion(): Promise<{ version: string }> {
 
 export const useGetAppVersion = () =>
   useQuery<{ version: string }>([], async () => getVersion(), {
-    refetchInterval: REFETCH_INTERVAL_ONE_HOUR,
-    refetchOnWindowFocus: false,
```

Comment:
The getVersion function does pop up a lot in sentry. I think because it's trying to access local files. With refetch on window focus we may get more of these errors

Reply:
could we avoid this by bundling the app version in the config file instead when we generate that locally?

Reply:
Isn't that better suited for a service worker? I belive my PWA PR adds this automatically.

Reply:
Yes indeed, let's discuss that in the PWA PR :)